kong/plugin_service_test.go

```diff
@@ -658,6 +658,39 @@ func TestFillPluginDefaults(T *testing.T) {
 				Enabled:   Bool(false),
 			},
 		},
+	}
+
+	for _, tc := range tests {
+		T.Run(tc.name, func(t *testing.T) {
+			p := tc.plugin
+			fullSchema, err := client.Plugins.GetFullSchema(defaultCtx, p.Name)
+			assert.NoError(err)
+			assert.NotNil(fullSchema)
+			if err := FillPluginsDefaults(p, fullSchema); err != nil {
+				t.Errorf(err.Error())
+			}
```

Comment:
That's an unfortunate issue with several tests that I've seen around here:

- using `assert` where we should use `require`
- using `assert` object based on the supertest's object: this will fail the whole test, while it should only fail the subtest

```suggestion
			require.NoError(t, err)
			require.NotNil(t, fullSchema)
			require.NoError(t, FillPluginsDefaults(p, fullSchema))
```

Reply:
Why resolved but not comment and/or changed 😢 ?